README.md

```diff
@@ -1,67 +1,143 @@
+
+Find out more about Sendbird Desk SDK for Android on [Desk SDK for Android doc](https://sendbird.com/docs/desk/v1/android/getting-started/about-desk-sdk). If you have any comments or questions regarding bugs and feature requests, visit [Sendbird community](https://community.sendbird.com).
+
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird Desk SDK for Android.
+
+### Requirements
+
+- `Android 4.0 (API level 14) or later`
+- `Java 7 or later`
+- `Gradle 3.4.0 or later`
+
+<br />
+
+## Getting started
+
+This section gives you information you need to get started with Sendbird Desk SDK for Android. 
+
+### Try the sample app
+
+Our sample app demonstrates the core features of Sendbird Desk SDK. Download the app from our GitHub repository to get an idea of what you can do with the actual SDK and to get started building your own project.
+ 
+- https://github.com/sendbird/quickstart-desk-android
+
+### Step 1: Create a Sendbird application from your dashboard
+
+A Sendbird application comprises everything required in a chat service including users, message, and channels. To create an application:
+
+1. Go to the Sendbird Dashboard and enter your email and password, and create a new account. You can also sign up with a Google account.
+2. When prompted by the setup wizard, enter your organization information to manage Sendbird applications.
+3. Lastly, when your dashboard home appears after completing setup, click **Create +** at the top-right corner.
+
+Regardless of the platform, only one Sendbird application can be integrated per app; however, the application supports communication across all Sendbird’s provided platforms without any additional setup. 
 
-Installing the Desk SDK is a straightforward process if you're familiar with using external libraries or SDKs in your projects.
-To install the Desk SDK using Gradle, add the following lines to your project-level `build.gradle` file.
-```gradle
+> **Note**: All the data is limited to the scope of a single application, thus users in different Sendbird applications are unable to chat with each other.
+
+### Step 2: Download and install the Desk SDK
+
+Installing the Desk SDK is simple if you’re familiar with using external libraries or SDKs in your projects. You can install using the following three methods. 
+
+#### - Project level
+
+```bash
 repositories {
     maven { url "https://raw.githubusercontent.com/sendbird/SendBird-SDK-Android/master/" }
     maven { url "https://raw.githubusercontent.com/sendbird/SendBird-Desk-SDK-Android/master/" }
 }
 ```
 
-And then add the following lines to your app-level `build.gradle` file.
-```gradle
+#### - App Level
+
+```bash
+// Add the following dependency to the build.gradle file at the app level.
 dependencies {
     implementation 'com.sendbird.sdk:sendbird-android-sdk:3.0.141'
     implementation 'com.sendbird.sdk:sendbird-desk-android-sdk:1.0.8'
```

Comment:
The latest versions for Messaging SDK and Desk SDK are `3.0.151`, `1.0.9` respectively.

Reply:
Done!